src/wam2layers/analysis/visualization.py

```diff
@@ -120,6 +119,7 @@ def visualize_input_data(config_file):
     out_dir = Path(config.output_folder) / "figures"
     out_dir.mkdir(exist_ok=True, parents=True)
     fig.savefig(out_dir / "input_event.png", dpi=200)
+    logger.info("Generate figure showing the cumulative moisture inputs.")
```

Comment:
```suggestion
    logger.info("Generated figure showing the cumulative moisture inputs.")
```
It could be nice to also include the output path, e.g. "Figure of cumulative outputs written to `path/to/figure`"

Reply:
Cool! The path information is added to the log info.